### ISSUE TITLE
Warning when inserting documents to collection

### ISSUE THREAD

**Issue by Arputikos** [milvus-io/milvus-haystack#35]
I created some documents and adding them to milvus zilliz cloud.
Got warnings:
Document f8a733afe670032c5f438ba53b7e0d64d4a405c8ad1751674eb72bbd7c6c12c5 has metadata fields with unsupported types: ['_split_overlap']. Supported types refer to Pymilvus DataType. The values of these fields will be discarded.

even though _split_overlap is not something that I added - it's from document splitter I guess. My pipeline is following:
DocumentCleaner
DocumentSplitter
DocumentEmbedder
DocumentWriter

**Comment by zc277584121** [comment]
@Arputikos yes, it's caused by  document splitter,  https://github.com/milvus-io/milvus-haystack/issues/32
can you give us some feedback, like whether there are too many warning logs to affect the viewing of other logs. If so, we will try to discard it. Any suggestion is appreciated

**Comment by zc277584121** [comment]
thanks, good suggestion, we will take consideration and try to meet it
